tests/e2e/stateapp/stateapp_test.go

```diff
@@ -608,38 +608,58 @@ func TestQueryStateStore(t *testing.T) {
 	_, err := utils.HTTPGetNTimes(externalURL, numHealthChecks)
 	require.NoError(t, err)
 
-	// Populate store.
-	url := fmt.Sprintf("%s/test/%s/%s/querystatestore", externalURL, "http", "load")
-	body, err := os.ReadFile("query-data/dataset.json")
-	require.NoError(t, err)
-
-	_, status, err := utils.HTTPPostWithStatus(url, body)
-	require.NoError(t, err)
-	require.Equal(t, 204, status)
-
-	tests := []struct {
-		path string
-		keys int
+	for _, store := range []struct {
+		name    string
+		isRedis bool
 	}{
 		{
-			path: "./query-data/query.json",
-			keys: 3,
+			name:    "querystatestore",
+			isRedis: false,
```

Comment:
Same comment as in the previous PR, please, just add the content type all the time and we can remove any redis specific reference here.